samples/k8s-batch-examples.md

```diff
@@ -15,675 +13,698 @@ os.environ['MESH_IP'] = MESH_IP
+        75,
+        39,
+        9,
+        44,
+        32,
+        97,
+        99,
+        40,
+        13,
+        27,
+        25,
+        36,
+        18,
+        77,
+        62,
+        60
+      ],
+      "datatype": "INT32",
+      "shape": [
+        1,
+        16
+      ]
+    },
+    {
+      "name": "INPUT1",
+      "data": [
+        39,
+        7,
+        14,
+        58,
+        13,
+        88,
+        98,
+        66,
+        97,
+        57,
+        49,
+        3,
+        49,
+        63,
+        37,
+        12
+      ],
+      "datatype": "INT32",
+      "shape": [
+        1,
+        16
+      ]
+    }
+  ]
+}
 
+```
 
 ```bash
 %%bash
 mlserver infer -u ${MESH_IP} -m tfsimple1 -i batch-inputs/tfsimple-input.txt -o /tmp/tfsimple-output.txt --workers 5 -b
 ```
+```
 
-    2022-11-16 18:26:08,522 [mlserver] INFO - Using asyncio event-loop policy: uvloop
-    2022-11-16 18:26:08,523 [mlserver] INFO - server url: 172.19.255.1
-    2022-11-16 18:26:08,523 [mlserver] INFO - model name: tfsimple1
-    2022-11-16 18:26:08,523 [mlserver] INFO - request headers: {}
-    2022-11-16 18:26:08,523 [mlserver] INFO - input file path: batch-inputs/tfsimple-input.txt
-    2022-11-16 18:26:08,523 [mlserver] INFO - output file path: /tmp/tfsimple-output.txt
-    2022-11-16 18:26:08,523 [mlserver] INFO - workers: 5
-    2022-11-16 18:26:08,523 [mlserver] INFO - retries: 3
-    2022-11-16 18:26:08,523 [mlserver] INFO - batch interval: 0.0
-    2022-11-16 18:26:08,523 [mlserver] INFO - batch jitter: 0.0
-    2022-11-16 18:26:08,523 [mlserver] INFO - connection timeout: 60
-    2022-11-16 18:26:08,523 [mlserver] INFO - micro-batch size: 1
-    2022-11-16 18:26:08,620 [mlserver] INFO - Finalizer: processed instances: 100
-    2022-11-16 18:26:08,620 [mlserver] INFO - Total processed instances: 100
-    2022-11-16 18:26:08,620 [mlserver] INFO - Time taken: 0.10 seconds
-
+```
+2022-11-16 18:26:08,522 [mlserver] INFO - Using asyncio event-loop policy: uvloop
+2022-11-16 18:26:08,523 [mlserver] INFO - server url: 172.19.255.1
+2022-11-16 18:26:08,523 [mlserver] INFO - model name: tfsimple1
+2022-11-16 18:26:08,523 [mlserver] INFO - request headers: {}
+2022-11-16 18:26:08,523 [mlserver] INFO - input file path: batch-inputs/tfsimple-input.txt
+2022-11-16 18:26:08,523 [mlserver] INFO - output file path: /tmp/tfsimple-output.txt
+2022-11-16 18:26:08,523 [mlserver] INFO - workers: 5
+2022-11-16 18:26:08,523 [mlserver] INFO - retries: 3
+2022-11-16 18:26:08,523 [mlserver] INFO - batch interval: 0.0
+2022-11-16 18:26:08,523 [mlserver] INFO - batch jitter: 0.0
+2022-11-16 18:26:08,523 [mlserver] INFO - connection timeout: 60
+2022-11-16 18:26:08,523 [mlserver] INFO - micro-batch size: 1
+2022-11-16 18:26:08,620 [mlserver] INFO - Finalizer: processed instances: 100
+2022-11-16 18:26:08,620 [mlserver] INFO - Total processed instances: 100
+2022-11-16 18:26:08,620 [mlserver] INFO - Time taken: 0.10 seconds
 
+```
 
 ```bash
 %%bash
```

Comment:
:information_source: This is another case of IPython magics causing issues.  A user can clearly see not to copy this, but it's still a bit unhelpful.  Much like the `%env` magic, we probably want to have a follow-up PR to handle cases like this.